docker/Dockerfile

```diff
@@ -8,9 +8,19 @@ RUN pip install awscli requests boto3 && mkdir -p /usr/local/share/thumbnailer
 COPY scripts/extract_video_thumbnail.sh /usr/local/bin/extract_video_thumbnail.sh
 COPY scripts/extract_audio_thumbnail.sh /usr/local/bin/extract_audio_thumbnail.sh
 COPY scripts/extract_image_thumbnail.sh /usr/local/bin/extract_image_thumbnail.sh
+COPY scripts/extract_raw_thumbnail.sh /usr/local/bin/extract_raw_thumbnail.sh
 COPY scripts/analyze_media_file.py /usr/local/bin/analyze_media_file.py
 COPY scripts/audio.gpi /usr/local/share/thumbnailer/audio.gpi
 COPY scripts/extract_thumbnail.sh /usr/local/bin/extract_thumbnail.sh
 RUN chmod a+x /usr/local/bin/extract_*.sh
+RUN yum -y groupinstall 'Development Tools'
```

Comment:
please could you separate the installation/compilation into a script, copy that to the image and run it; at the end of the script uninstall the development tools and wget.
The reason for this is that each of these lines adds a layer to the docker image, so removing the tools in the dockerfile won't actually reduce the size of the image.  Having all of the development tools loaded onto the image as well is a little excessive.

Reply:
Done.